includes/sql-schema/update.php

```diff
@@ -108,10 +96,5 @@
         echo Artisan::output();
     }
 
-    if (isset($schemaLock)) {
-        $schemaLock->release();
-    }
-} catch (LockException $e) {
-    echo $e->getMessage() . PHP_EOL;
-    $return = 1;
+    $schemaLock->release();
```

Comment:
Wouldn't this one be released even if skip =true?

Reply:
You can't release a lock you don't own unless you forceRelease()